crates/ruff_python_ast/generate.py

```diff
@@ -174,7 +174,8 @@ def write_owned_enum(out: list[str], groups: list[Group]) -> None:
         """)
         for node in group.nodes:
             out.append(
-                f"""{group.owned_enum_ty}::{node.variant}(node) => node.visit_source_order(visitor),"""
+                f"{group.owned_enum_ty}::{node.variant}(node) => "
+                "node.visit_source_order(visitor),"
```

Comment:
Splitting this over multiple line decreases readability. I suggest disabling E501 and keep it as regular string.